### PR TITLE
test(integration): Enable Integration test

### PR DESCRIPTION
**Pull request by Khaledgarbaya**

### CODE REVIEW

**Comment by geigerzaehler** [comment]
We should run integration tests only for one node version

**Comment by Khaledgarbaya** [comment]
yeah that's done already https://github.com/contentful/contentful-sdk-core/blob/master/bin/test-ci.sh#L19

File: test/integration/integration-tests.js [hunk 1]
```diff
@@ -111,8 +107,8 @@ test('Create space for tests which create, change and delete data', (t) => {
     entryWriteTests(t, space)
     assetWriteTests(t, space)
     webhookTests(t, space)
-    spaceMembershipTests(t, space)
-    roleTests(t, space)
+    // spaceMembershipTests(t, space)
+    // roleTests(t, space)
```
Comment (axe312ger) [inline test/integration/integration-tests.js]: why? Fix it or kill it? ^^

Comment (Khaledgarbaya) [inline test/integration/integration-tests.js]: I'm trying to find which test is hitting the limits

**Comment by axe312ger** [comment]
@Khaledgarbaya should we close here?

Fixed by https://github.com/contentful/contentful-management.js/pull/127/commits/053c453f9f2a510f67930c33433972f84539dfdf and https://github.com/contentful/contentful-management.js/pull/127/commits/16a80af7fb52e434b908c8bcc9a6acb1b30f297e

**Comment by Khaledgarbaya** [comment]
@axe312ger got for it

**Pull request closed without merging**